### PR TITLE
Added Couchbase community edition chart

### PR DESCRIPTION
**Pull request by algchoo**
* added chart to deploy a statefulset and service
* added readme with instructions on how to configure nodes and create a cluster

### CODE REVIEW

File: charts/couchbase-community/README.md [hunk 1]
```diff
@@ -0,0 +1,33 @@
...
+```
+kubectl port-forward pod/couchbase-community-0 8091
+```
+
+2. Run the following command to get the IP addresses of the individual pods. Note the IP addresses for the second and third pods. 
+```
+kubectl get pod -o wide
+```
+
+3. Using your browser of choice, visit `localhost:8091` and you will see the login page for the node. The username and password will be `Administrator:password` for all nodes.
```
Comment (jsirianni) [inline charts/couchbase-community/README.md:27]: I had to create a password here. The default username was filled in as `Administrator`

File: charts/couchbase-community/README.md [hunk 2]
```diff
@@ -0,0 +1,33 @@
...
+
+4. After a successful login, go to the **Servers** page
```
Comment (jsirianni) [inline charts/couchbase-community/README.md:29]: Before this happens, I had to configure a "new cluster"

Screenshot from 2023-06-16 16-11-51

**Review by jsirianni — CHANGES_REQUESTED** [review]
Working well but I think the doc is missing a few details when deploying a new cluster.

**Comment by algchoo** [comment]
> Working well but I think the doc is missing a few details when deploying a new cluster.

Regarding the issues we we're seeing on your end, I was able to run this in k8s in various ways. Like you said, if it's working consistently for me, we should probably keep it.
* colima
* minikube using the docker driver
* minikube using the qemu2 driver

**Review by jsirianni — APPROVED** [review]

**Pull request merged**